app/styles/scss/modules/_test-block.scss

```diff
@@ -0,0 +1,130 @@
+/*   footer is located at the bottom   */
```

Comment:
Because this style for my mockup with header and footer
But now our main version has another style, so I temporary commented my style